src/model/presentation.js

```diff
@@ -48,10 +48,12 @@ class Presentation {
 
     const allowedMediaUploads = this.getAllowedMediaUploads();
     const groupedTags = this.getAllowedTags();
+    const showSpeakers = this._presentation.type?.use_speakers && this._presentation.type?.max_speakers > 0;
```

Comment:
i do think that this._presentation.type?.use_speakers is just enough
max_speakers could be equal to zero , thats mean infinite speakers ( not upper limit)
regards